docs/source/create_channel/create_channel.md

```diff
@@ -0,0 +1,383 @@
+```
+
+The command above provides the path to the channel creation artifact using the `-f` flag and uses the `-c` flag to specify the channel name. The `-o` flag is used to select the ordering node that will be used to create the channel. The `--cafile` is the path to the TLS certificate of the ordering node. When you run the `peer channel create` command, the `peer` CLI will generate the following response:
+```
+2020-03-06 17:33:49.322 EST [channelCmd] InitCmdFactory -> INFO 00b Endorser and orderer connections initialized
+2020-03-06 17:33:49.550 EST [cli.common] readBlock -> INFO 00c Received block: 0
+```
+Because we are using a RAFT ordering service, you may get some status unavailable messages that you can safely ignore. The command will return the genesis block of the new channel to the location specified by the `--outputBlock` flag.
+
+## Join peers to the channel
+
+After the channel has been created, we can join the channel with our peers. Organizations that are members of the channel can fetch the channel genesis block from the ordering service using the [peer channel fetch](../commands/peerchannel.html#peer-channel-fetch) command. The organization can then use the genesis block to join the peer to the channel using the [peer channel join](../commands/peerchannel.html#peer-channel-join) command. Once the peer is joined to the channel, the peer will build the blockchain ledger by retrieving the other blocks on the channel from the ordering service.
+
+Since we are already operating the `peer` CLI as the Org1 admin, let's join the Org1 peer to the channel. Since Org1 submitted the channel creation transaction, we already have the channel genesis block on our file system. Join the Org1 peer to the channel using the command below.
+```
+peer channel join -b ./channel-artifacts/channel1.block
+```
+
+The `CORE_PEER_ADDRESS` environment variable has been set to target ``peer0.org1.example.com``. A successful command will generate a response from ``peer0.org1.example.com`` joining the channel:
+```
+2020-03-06 17:49:09.903 EST [channelCmd] InitCmdFactory -> INFO 001 Endorser and orderer connections initialized
+2020-03-06 17:49:10.060 EST [channelCmd] executeJoin -> INFO 002 Successfully submitted proposal to join channel
+```
+
+You can verify that the peer has joined the chanel using the [peer channel getinfo](../commands/peerchannel.html#peer-channel-getinfo) command:
+```
+peer channel getinfo -c channel1
+```
+The command will list the block height of the channel and the hash of the most recent block. Because the genesis block is the only block on the channel, the height of the channel will be 1:
+```
+2020-03-13 10:50:06.978 EDT [channelCmd] InitCmdFactory -> INFO 001 Endorser and orderer connections initialized
+Blockchain info: {"height":1,"currentBlockHash":"kvtQYYEL2tz0kDCNttPFNC4e6HVUFOGMTIDxZ+DeNQM="}
+```
+
```

Comment:
Might be worth noting here that adding a peer to a channel does not change the block height as it's not a channel config update operation.

Reply:
Added this to the end of the section:

You can use the `peer channel getinfo` command once again to confirm that the Org2 peer is joined to the channel.
```
peer channel getinfo -c channel1
```
Because you do not have to update the channel configuration to join a peer to the channel, the genesis block is still the only block in the channel and the block height does not change.

If it is too obvious, and does not need to be cleared up, we can always remove.